packages/apps/shopify-app-react-router/package.json

```diff
@@ -102,18 +90,13 @@
   },
   "peerDependencies": {
     "react-router": "^7.6.2",
-    "@shopify/polaris": "*",
-    "react": "*"
-  },
-  "peerDependenciesMeta": {
-    "@shopify/polaris": {
-      "optional": true
-    }
+    "react": ">=18",
```

Comment:
Do we still require react as a peer dependency? Or is react-router specifying it as a peer dependency enough?

Reply:
I removed this at one point and the RR app wouldn't build, so I think it's needed.